src/libponyc/pass/docgen.c

```diff
@@ -403,6 +407,30 @@ static void doc_type_list(docgen_t* docgen, ast_t* list, const char* preamble,
   fprintf(docgen->type_file, "%s", postamble);
 }
 
+static bool is_for_testing(const char* name, ast_t* list)
+{
+  assert(name != NULL);
+  assert(list != NULL);
+
+  if (strncmp(name, "_Test", 5) == 0) return true;
+
+  if(ast_id(list) == TK_NONE)
+    return false;
+
+  for(ast_t* p = ast_child(list); p != NULL; p = ast_sibling(p))
+  {
+    if (ast_id(p) == TK_NOMINAL)
+    {
+      AST_GET_CHILDREN(p, package, id, tparams, cap, ephemeral);
```

Comment:
So I need: `AST_GET_CHILDREN(p, package, id)`?

I'm still not getting it. I have an AST node `p` and I want its id...

What do I do?